frontend/src/components/contacts.tsx

```diff
@@ -1,57 +1,59 @@
-import React, { useState } from "react";
-import axios from "axios";
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
same - really justify to yourself why this should be ignored

Reply:
rm'ed